view/graph/GraphArea.kt

```diff
@@ -35,8 +35,6 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.ExperimentalComposeUiApi
```

Comment:
Drawing text in Compose is much more expensive than drawing polygons. So we need to be more aggressive in choosing which texts to actually draw. Clearly, any text that we know to be fully outside the viewport should not be drawn.

We aim to ensure the initial "explosion" is as smooth as possible, and the resulting stable visualisation as rich as possible. The variables to consider are: total vertices/edges in graph; total vertices/edges in viewport; viewport scale; physics alpha.